Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -6467,6 +6480,48 @@ static NSTextAlignment nsTextAlignmentFromTextAlignment(TextAlignment textAlignm
 
 #endif // ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
 
+#if ENABLE(UNIFIED_TEXT_REPLACEMENT)
+void WebViewImpl::willBeginTextReplacementSession(const WTF::UUID& uuid, CompletionHandler<void(const Vector<WebUnifiedTextReplacementContextData>&)>&& completionHandler)
+{
+    m_page->willBeginTextReplacementSession(uuid, WTFMove(completionHandler));
```

Comment:
Please add a `protectedPage()` getter that returns a `Ref<WebPageProxy` and call that. Same for other functions below.

Reply:
@cdumez like this?

```
Ref< WebPageProxy > WebViewImpl::protectedPage() const
{
    return m_page.get();
}
```

I’m also wondering why `m_page` seems to be ok to be used in other parts of this file?